app/styles/components/_detail-learning-materials.scss

```diff
@@ -1,11 +1,23 @@
-.detail-learning-materials {
-  .learning-material-actions {
-    @include span-columns(3 of 12);
-    @include omega;
-    float: right;
+.detail-block {
```

Comment:
I changed it to `detail-container`. Since couple of `detail-container`'s child elements are absolutely positioned, `detail-container` needs to have a `relative` position as a parent. It can't apply to the component class.